src/SelectableGroup.tsx

```diff
@@ -361,7 +361,10 @@ class SelectableGroup extends Component<TSelectableGroupProps> {
       return null
     }
 
-    const isCollided = doObjectsCollide(selectboxBounds, item.bounds!, tolerance, this.props.delta)
+    let isCollided = false;
+    for (let i = 0; i < item.bounds.length; i+=1) {
+        isCollided = doObjectsCollide(selectboxBounds, item.bounds[i], tolerance, this.props.delta);
+    }
```

Comment:
We should check here if some of the `item.bounds[]` collides with `selectboxBounds`. 
Looks like if the last bound from `item.bounds` doesn't collide with `selectboxBounds` `isCollided` will be set to `false` even if some other item bound collides with selectbox which is wrong.

It can be fixed with `[].some(fn)` method. 

```ts
const isCollided = item.bounds.some(bound => doObjectsCollide(selectboxBounds, bound, tolerance, this.props.delta))
```